sdk/cosmos/azure_data_cosmos/examples/cosmos/create.rs

```diff
@@ -32,6 +32,10 @@ pub enum Subcommands {
         /// The JSON of the new item.
         #[clap(long, short)]
         json: String,
+
+        /// If set, the updated item will be included in the response.
+        #[clap(long)]
```

Comment:
Just occurred to me: wait, we're using an old enough clap that it's still using `#[clap]` instead of `#[arg]`? Or did they retain support for backcompat? `#[arg]` is current.

Also, if compatible with `#[arg]`, I thought for sure this required `#[arg(action = ArgAction::SetTrue)]`.

Reply:
>  wait, we're using an old enough clap that it's still using #[clap] instead of #[arg]? Or did they retain support for backcompat? #[arg] is current.

Just a habit on my end. It seems to have been retained for compat, but our version of clap absolutely supports `#[arg]`/`#[command]`.

> Also, if compatible with #[arg], I thought for sure this required #[arg(action = ArgAction::SetTrue)].

It's implied when you have a `bool` arguments: https://docs.rs/clap/latest/clap/_derive/index.html#arg-types